### DISCUSSION TITLE
[REQUEST] GET_MERCHANTS_LISTINGS_FYP_REPORT documentation

### DISCUSSION THREAD

**Discussion by therubioftw** [amzn/selling-partner-api-models#3345] [General]
Hello everybody!

It seems that the FYP_Reports is becoming the main way for staying on top of listing issues, so I've pulled it on all of our clients.
For a field called `status`, I've gotten the following values: `At risk, Blocked, Detail Page Removed, Search Suppressed, fyp-atrisk-label, fyp-search-suppressed-filter-label`.

My issue is that there is just about 0 documentation about this - so I'm not sure I understand what exactly the difference is between all of these, or what might have caused them.

Help?

**Comment by therubioftw** [comment]
@jevoniuk We moved to this report following [your announcement](https://github.com/amzn/selling-partner-api-docs/discussions/2348) - if there's anything you can do to facilitate this, I know it will be highly appreciated :)

It's extremely hard to follow these reports with no documentation...

**Comment by moltar** [comment]
We would like more information too.

Notably, it appears that there can be multiple rows per supression. But this is not documented anywhere.

Please describe what it does and how to use it.

**Comment by contrebande-labs** [comment]
Actually, any schema or model for any report (especially the new JSON ones) would be highly appreciated.

**Reply by contrebande-labs** [reply]
For anyone reading this, following the October 2022 announcement, they seem to have added just that : https://github.com/amzn/selling-partner-api-models/tree/main/schemas

**Reply by therubioftw** [reply]
Thanks! Though there still isn't much there, we'll see how fast they catch up to all the report types :)
